lib/alternatives.js

```diff
@@ -1,45 +1,39 @@
-'use strict';
+"use strict";
```

Comment:
You seem to have changed the quote style that the original library was using. I don't know if that is wise here, as it will create changes every file you touch.